pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -360,26 +396,40 @@ message Capability {
     repeated string drop_capabilities = 2;
 }
 
-// LinuxContainerConfig contains platform-specific configuration for
-// Linux-based containers.
-message LinuxContainerConfig {
-    // Resources specification for the container.
-    optional LinuxContainerResources resources = 1;
+// SecurityContext holds security configuration that will be applied to a container.
+// Some fields are present in both SecurityContext and PodSecurityContext.  When both
+// are set, the values in SecurityContext take precedence.
+message SecurityContext {
     // Capabilities to add or drop.
-    optional Capability capabilities = 2;
+    optional Capability capabilities = 1;
+    // If set, run container in privileged mode.
+    // Processes in privileged containers are essentially equivalent to root on the host.
+    optional bool privileged = 2;
     // Optional SELinux context to be applied.
     optional SELinuxOption selinux_options = 3;
-    // User contains the user for the container process.
-    optional LinuxUser user = 4;
+    // The UID to run the entrypoint of the container process.
+    // Defaults to user specified in image metadata if unspecified.
+    // May also be set in PodSecurityContext.  If set in both SecurityContext and
+    // PodSecurityContext, the value specified here takes precedence.
+    optional int64 run_as_user = 4;
+    // Indicates that the container must run as a non-root user.
```

Comment:
Is the runtime supposed to pick a non root uid if this is set or is run_as_user also expected to be set when this is set?